api/utils/cluster.go

```diff
@@ -0,0 +1,30 @@
+/*
+Copyright 2021 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package utils
+
+import (
+	"encoding/hex"
+	"fmt"
+
+	"github.com/gravitational/teleport/api/constants"
+)
+
+// EncodeClusterName encodes cluster name in the SNI hostname
+func EncodeClusterName(clusterName string) string {
```

Comment:
yes I was aware about this function but the `api/client` package should not depend on `teleport/lib/` package because that will lead to cyclomatic dependency. Right now `api/client` package depends only on external libraries thus I thought that 
 "A little copying is better than a little dependency." -  but don't have strong opinion and I can move all `EncodeClusterName` `DecodeClusterName`to the api/client and add api/client in places where those functions are used.

Reply:
@smallinsky Yeah, I think it's worthwhile to move those in this case (or at least type alias them). Teleport already depends on the api module so it wouldn't be a new dependency.